recipes/pymol-open-source/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "2.4.0" %}
+{% set number = "0" %}
+{% set sha256 = "5ede4ce2e8f53713c5ee64f5905b2d29bf01e4391da7e536ce8909d6b9116581" %}
+
+package:
+  name: pymol-open-source
+  version: {{ version }}
+
+source:
+  url: https://github.com/schrodinger/pymol-open-source/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ number }}
+  script: "{{ PYTHON }} setup.py install --use-msgpackc=no --no-libxml --no-vmd-plugins"
+  skip: True  # [py < 37]
+  skip: True  # [osx]
```

Comment:
How come macOS is being skipped? (it's unusual for Linux and Windows to work but not macOS)

Reply:
On MacOS I encountered a quite confusing compiler error, which I was notable to resolve. Therefore, I excluded MacOS for now.